### PR TITLE
Issue 133 Headings in templates & guide should be sentence case

### PR DESCRIPTION
**Pull request by emmurphy1**

### CODE REVIEW

**Review by sterobin — APPROVED** [review]
Yaaaay! Thank you so much, @emmurphy1 So glad we finally broke down and got that addressed.

File: modular-docs-manual/content/topics/module_auditing.adoc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [id="auditing"]
-= Auditing Your Feature-Based Manual
+= Auditing Your feature-based manual
```
Comment (mjahoda) [inline modular-docs-manual/content/topics/module_auditing.adoc:2]: Your -> your

**Review by mjahoda — APPROVED** [review]
Except for one omission (I put a comment inline), it looks great to me. Thank you for this significant and long-term postponed change!

**Comment by vsodhi** [comment]
LGTM.

**Pull request merged**